cognite/neat/_session/_prepare.py

```diff
@@ -507,65 +420,25 @@ def to_data_product(
             include: The views to include in the data product data model. Can be either "same-space" or "all".
                 If you set same-space, only the views in the same space as the data model will be included.
         """
-        source_id, rules = self._state.data_model.last_verified_dms_rules
-
-        dms_ref: DMSRules | None = None
-        view_ids, container_ids = DMSValidation(rules, self._client).imported_views_and_containers_ids()
-        if view_ids or container_ids:
-            if self._client is None:
-                raise NeatSessionError(
-                    "No client provided. You are referencing unknown views and containers in your data model, "
-                    "NEAT needs a client to lookup the definitions. "
-                    "Please set the client in the session, NeatSession(client=client)."
-                )
-            schema = self._client.schema.retrieve([v.as_id() for v in view_ids], [c.as_id() for c in container_ids])
-
-            importer = DMSImporter(schema)
-            reference_rules = importer.to_rules().rules
-            if reference_rules is not None:
-                imported = VerifyDMSRules("continue").transform(reference_rules)
-                if dms_ref := imported.rules:
-                    rules = rules.model_copy(deep=True)
-                    if rules.containers is None:
-                        rules.containers = dms_ref.containers
-                    else:
-                        existing_containers = {c.container for c in rules.containers}
-                        rules.containers.extend(
-                            [c for c in dms_ref.containers or [] if c.container not in existing_containers]
-                        )
-                    existing_views = {v.view for v in rules.views}
-                    rules.views.extend([v for v in dms_ref.views if v.view not in existing_views])
-                    existing_properties = {(p.view, p.view_property) for p in rules.properties}
-                    rules.properties.extend(
-                        [p for p in dms_ref.properties if (p.view, p.view_property) not in existing_properties]
-                    )
-
-        start = datetime.now(timezone.utc)
-        transformer = ToExtension(
-            new_model_id=data_model_id,
-            org_name=org_name,
-            type_="data_product",
-            include=include,
-        )
-        output = transformer.transform(rules)
-        end = datetime.now(timezone.utc)
-
-        change = Change.from_rules_activity(
-            output,
-            transformer.agent,
-            start,
-            end,
-            (
-                f"Prepared data model {data_model_id} to be data product model "
-                f"on top of {rules.metadata.as_data_model_id()}"
+        if self._client is None:
+            raise NeatSessionError(
```

Comment:
same here, but since we are always using "client" in the tutorials, maybe it is better to be consistent with that name